Xamarin.Forms.Core/DeviceInfo.cs

```diff
@@ -2,17 +2,18 @@
 using System.ComponentModel;
 using System.Runtime.CompilerServices;
 
-namespace Xamarin.Forms
+namespace Xamarin.Forms.Internals
 {
-	internal abstract class DeviceInfo : INotifyPropertyChanged, IDisposable
+	public abstract class DeviceInfo : INotifyPropertyChanged, IDisposable
```

Comment:
Yeah, good idea. Make 'em even harder to bind to. Let's do that in a separate change PR right after we merge this one.